frontend/src/pages/Projects/Project/Billing/FormConfig.ts

```diff
@@ -164,8 +164,10 @@ export const FormConfig = (query: UseQueryResult<AxiosResponse, unknown>) => {
           );
           return (
             //parseFloat with regex converts the money formatted string "$100.00" to a number like "100.00.  This is required to do a compare."
-            parseFloat((value as string).replace(/[^0-9.]/g, "")) <=
-            parseFloat(recoveredBudget.replace(/[^0-9.]/g, ""))
+
+            parseFloat(
+              recoveredBudget !== "" ? (recoveredBudget as string).replace(/[^0-9.]/g, "") : "0"
+            ) <= parseFloat((value as string).replace(/[^0-9.]/g, ""))
```

Comment:
I think this will cause negative amounts to become positive since it will replace the `-`.